airsonic-main/src/main/java/org/airsonic/player/controller/api/IndexRESTController.java

```diff
@@ -0,0 +1,21 @@
+package org.airsonic.player.controller.api;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+@RequestMapping(value = "/api")
```

Comment:
I would suggest prefixing the whole API with `/api/v1`. It's a common pattern versioning APIs so we're able to cleanly phase them out if needed, making backwards-incompatible changes somewhat less likely. We could also forward `/api/legacy` or `/api/compat` to the Subsonic-compatible API currently available at `/rest`.

```suggestion
@RequestMapping(value = "/api/v1")
```